OoyalaSkinSampleApp/app/src/main/java/com/ooyala/sample/players/AbstractHookActivity.java

```diff
@@ -215,7 +223,35 @@ protected Options getOptions(){
 	  else if(hevcMode.equalsIgnoreCase("HEVCNotPreferred")) {
 		  optionBuilder.enableHevc(false);
 	  }
+	  if (!markersFileName.isEmpty()) {
+	  	  try {
+			  optionBuilder.setMarkers(loadJSONFromAsset(markersFileName));
+		  }  catch (JSONException e) {
+              Log.e(TAG, "Exception Thrown while json file loading from assets", e);
+	  	  }
+	  }
 	  Options options =  optionBuilder.build();
 	  return options;
   }
+
+	/**
+	 *
+	 * @param name of asset file in a JSON format
+	 * @return asset file as JSON object
+	 */
+	public JSONObject loadJSONFromAsset(String name) throws JSONException {
```

Comment:
could you please remove the same code in `AssetActivity` that extends `AbstractHookActivity`

Reply:
Done